oak-store-spi/pom.xml

```diff
@@ -26,7 +26,11 @@
             <instructions>
               <Export-Package>
                 org.apache.jackrabbit.oak.jcr,
-                org.apache.jackrabbit.oak.jcr.observation.filter
+                org.apache.jackrabbit.oak.jcr.observation.filter,
+                org.apache.jackrabbit.oak.json,
+                org.apache.jackrabbit.oak.spi.cluster,
+                org.apache.jackrabbit.oak.spi.commit,
+                org.apache.jackrabbit.oak.spi.state
```

Comment:
right it looks like we export everything, but there's a blacklist in oak-parent that 'unexports' some packages: https://github.com/mreutegg/jackrabbit-oak/blob/trunk/oak-parent/pom.xml#L150